.github/workflows/linters.yml

```diff
@@ -0,0 +1,34 @@
+name: run linters
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  lint:
+    env:
+      CONAN_YAMLLINT_WERR: 1
+      CONAN_PYLINT_WERR: 1
+    strategy:
+      fail-fast: false
+      matrix:
+        hook: ["yaml_linter", "recipe_linter"]
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2.2.2
+        with:
+          python-version: "3.8"
+
+      - run: "pip3 install conan yamllint packaging pylint==2.10.2 astroid"
+
+      - name: install hook
+        run: |
+          conan config install https://github.com/conan-io/hooks.git
+          conan config set hooks.${{ matrix.hook }}
+
+      - name: run lint
+        run:  |
+          echo "::add-matcher::${{ matrix.hook }}.json"
+          python3 runlint.py ${{ github.event.pull_request.number }}
```

Comment:
on event `wokflow_dispatch` the PR number is still available? I dont see it in the docs https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#workflow_dispatch

I though an input value would be required 🤔